File: Dev/Code/Community.Entity/Model/OrderDetail/OrderDetailEntity.cs
using System;
using Community.Entity.Model.Order;
using Community.Entity.Model.Product;
using YooPoon.Core.Data;

namespace Community.Entity.Model.OrderDetail
{
	public class OrderDetailEntity : IBaseEntity
	{
		/// <summary>
		/// ID
		/// </summary>
		public virtual int Id { get; set; }
		/// <summary>
		/// 商品ID
		/// </summary>
		public virtual ProductEntity Product { get; set; }
		/// <summary>
		/// 商品名字
		/// </summary>
		public virtual string ProductName { get; set; }
		/// <summary>
		/// 单价
		/// </summary>
		public virtual decimal UnitPrice { get; set; }
		/// <summary>
		/// 数量
		/// </summary>
		public virtual decimal Count { get; set; }
		/// <summary>
		/// 商品页面快照
		/// </summary>
		public virtual string Snapshoturl { get; set; }
		/// <summary>
		/// 商品备注
		/// </summary>
		public virtual string Remark { get; set; }
		/// <summary>
		/// 添加人员
		/// </summary>
		public virtual int Adduser { get; set; }
		/// <summary>
		/// 添加时间
		/// </summary>
		public virtual DateTime Adddate { get; set; }
		/// <summary>
		/// 修改人
		/// </summary>
		public virtual int Upduser { get; set; }
		/// <summary>
		/// 修改时间
		/// </summary>
		public virtual DateTime Upddate { get; set; }
		/// <summary>
		/// 总价
		/// </summary>
		public virtual decimal Totalprice { get; set; }
		/// <summary>
		/// 订单主体
		/// </summary>
		public virtual OrderEntity Order { get; set; }
	}
}

File: Dev/Android/YoopoonMarket/src/com/yoopoon/market/domain/OrderDetailEntity.java
package com.yoopoon.market.domain;

import java.io.Serializable;

public class OrderDetailEntity implements Serializable {
	/**
	 * @fieldName: serialVersionUID
	 * @fieldType: long
	 * @Description: TODO
	 */
	private static final long serialVersionUID = 1L;
	public int Id = 1;
	public String No = null;
	public String ProductName;
	public float UnitPrice;
	public float Price = 0;
	public int Count;
	public String Snapshoturl;
	public String Remark;
	public int Adduser = 0;
	public String Adddate;
	public int Upduser = 0;
	public String Upddate;
	public float Totalprice;
	public ProductEntity Product;

	@Override
	public String toString() {
		return "OrderDetailEntity [Id=" + Id + ", No=" + No + ", ProductName=" + ProductName + ", UnitPrice="
				+ UnitPrice + ", Price=" + Price + ", Count=" + Count + ", Snapshoturl=" + Snapshoturl + ", Remark="
				+ Remark + ", Adduser=" + Adduser + ", Adddate=" + Adddate + ", Upduser=" + Upduser + ", Upddate="
				+ Upddate + ", Totalprice=" + Totalprice + ", Product=" + Product + "]";
	}
}
